packages/react/src/VideoPlayer/hooks/useIsElementFullScreen.ts

```diff
@@ -9,6 +9,12 @@ export const useIsElementFullScreen = (element?: HTMLElement | null): [boolean,
     const handleFullScreenChange = () => {
       const isCurrentlyFullScreen = document.fullscreenElement === element
 
+      if (isCurrentlyFullScreen) {
+        element.setAttribute('data-is-fullscreen', 'true')
```

Comment:
It seems like this is only used for styling purposes, is that right? If so, could you do this with a css class instead of a data attribute?

Reply:
Yes no problem. I did consider that but didn't know if the data attribute would be preferable to having to use a `:global` CSS selector, but I'll switch it out now